src/main/deploy/pathplanner/paths/go back to speaker from first note.path

```diff
@@ -0,0 +1,60 @@
+{
+  "version": 1.0,
+  "waypoints": [
+    {
+      "anchor": {
+        "x": 2.9029435836674633,
+        "y": 5.545677758026123
+      },
+      "prevControl": null,
+      "nextControl": {
+        "x": 3.951344791580537,
+        "y": 5.0214771540695695
+      },
+      "isLocked": true,
+      "linkedName": "first note"
```

Comment:
I'm assuming you're using a GUI to generate files like this. Maybe add a comment somewhere in the code (`RobotContainer.java` would be fine) to point people in the right direction when adding/modifying paths?